### ISSUE TITLE
CircleCI publishing macOS binary wheels fails

### ISSUE THREAD

**Issue by hacklschorsch** [LeastAuthority/python-challenge-bypass-ristretto#64]
macOS (and probably also all other) wheels aren't being published:

https://app.circleci.com/pipelines/github/LeastAuthority/python-challenge-bypass-ristretto/313/workflows/9b6b7c89-6153-4d1b-a760-61f1f14a0b2c/jobs/1179

> Package index credentials unavailable; skipping upload.

**Comment by hacklschorsch** [comment]
Maybe they are only published when CI runs on the `master` branch - and there it fails because CircleCI can't check out via SSH?

i.e. https://app.circleci.com/pipelines/github/LeastAuthority/python-challenge-bypass-ristretto/318/workflows/b226ee65-4018-4a4b-9a25-f3a0c193aa89/jobs/1234

```
Writing SSH key for checkout to "/Users/distiller/.ssh/id_rsa"
Writing SSH public key for checkout to "/Users/distiller/.ssh/id_rsa.pub"
Cloning git repository - git@github.com:LeastAuthority/python-challenge-bypass-ristretto.git
Cloning into '.'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```

**Comment by hacklschorsch** [comment]
Fixing this (the publishing to PyPI) is going to fix #57 and https://github.com/PrivateStorageio/ZKAPAuthorizer/issues/472

**Comment by hacklschorsch** [comment]
This is working as intended - only builds of the master branch should be uploaded
This is currently also broken and tracked in #67.
